server/etcdserver/apply/uber_applier.go

```diff
@@ -133,6 +133,31 @@ func (a *uberApplier) dispatch(r *pb.InternalRaftRequest) *Result {
 		}
 	}(time.Now())
 
+	switch {
+	case r.ClusterVersionSet != nil:
+		op = "ClusterVersionSet" // Implemented in 3.5.x
+		a.applyV3.ClusterVersionSet(r.ClusterVersionSet, shouldApplyV3)
+		return ar
+	case r.ClusterMemberAttrSet != nil:
+		op = "ClusterMemberAttrSet" // Implemented in 3.5.x
+		a.applyV3.ClusterMemberAttrSet(r.ClusterMemberAttrSet, shouldApplyV3)
+		return ar
+	case r.DowngradeInfoSet != nil:
+		op = "DowngradeInfoSet" // Implemented in 3.5.x
+		a.applyV3.DowngradeInfoSet(r.DowngradeInfoSet, shouldApplyV3)
+		return ar
+	case r.DowngradeVersionTest != nil:
+		op = "DowngradeVersionTest" // Implemented in 3.6 for test only
+		// do nothing, we are just to ensure etcdserver don't panic in case
+		// users(test cases) intentionally inject DowngradeVersionTestRequest
+		// into the WAL files.
+		return ar
+	default:
+	}
+	if !shouldApplyV3 {
+		return ar
```

Comment:
to align with orginal `applyInternalRaftRequest` function, we should return nil here